packages/dds/tree/package.json

```diff
@@ -85,7 +85,6 @@
     "@fluid-tools/build-cli": "^0.6.0-109663",
     "@fluidframework/build-common": "^1.1.0",
     "@fluidframework/build-tools": "^0.6.0-109663",
-    "@fluidframework/common-definitions": "^0.20.1",
```

Comment:
Just cleanup, right? Only the `dependencies` entry is needed?

Reply:
Yeah policy-check caught this one.